core/pom.xml

```diff
@@ -69,16 +69,6 @@
     <dependency>
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-client</artifactId>
-      <exclusions>
```

Comment:
Here, and in `launcher/pom.xml`, I moved the exclusions to the root POM instead of defining them here.